code/modules/mob/living/living_defense.dm

```diff
@@ -481,7 +481,6 @@
 
 /mob/living/proc/reagent_permeability()
 	return 1
-	return round(FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE*(fire_stacks/FIRE_MAX_FIRESUIT_STACKS)**2)

```

Comment:
This might be responsible for the bug where it is possible to apply reagents in certain ways through any amount of protection, including voidsuits?

Reply:
A quick look at the code suggests that human mobs use an overrided version of that proc, so if there is an issue, I don't think it's due to the `/mob/living` level proc.
https://github.com/PolarisSS13/Polaris/blob/9942d9cb25f0f9128880cf540646d8e833411e46/code/modules/mob/living/carbon/human/human_defense.dm#L561